video/out/d3d11/context.c

```diff
@@ -190,10 +191,26 @@ static int d3d11_color_depth(struct ra_swapchain *sw)
 {
     struct priv *p = sw->priv;
 
-    if (!p->backbuffer)
+    if (!p->swapchain) {
+        MP_ERR(sw->ctx, "Swap chain color depth was requested before it was "
+                        "fully initialized!\n");
+        return 0;
+    }
```

Comment:
This check seems unnecessary. `p->swapchain` is initialized in `d3d11_init` so it's always set when callers have access to the ra_swapchain methods. The other ra_swapchain methods just assume it's set.

Reply:
Fixed.

I think I was looking at the fullscreen call-back where a check was added ages ago when I was writing this, as it was called from a `ra_ctx` function pointer as opposed to a swap chain one.

Not sure if that one was necessary or not, but `d3d11_control` at least could be possibly utilized before `d3d11_init` was called. Will probably check later to verify.